playbooks/roles/journals/meta/main.yml

```
@@ -59,3 +59,4 @@ dependencies:
     edx_django_service_allow_cors_headers: '{{ JOURNALS_ALLOW_CORS_HEADERS }}'
     edx_django_service_max_webserver_upload: 10
     edx_django_service_allow_cors_credentials: '{{ JOURNALS_ALLOW_CORS_CREDENTIALS }}'
+    edx_django_service_enable_newrelic_distributed_tracing: '{{ JOURNALS_API_ENABLE_NEWRELIC_DISTRIBUTED_TRACING }}'
```

Perhaps this shouldn't start with ```JOURNALS_API``` since none of the other JOURNALS variables start with ```JOURNALS_API```
```suggestion
    edx_django_service_enable_newrelic_distributed_tracing: '{{ JOURNALS_ENABLE_NEWRELIC_DISTRIBUTED_TRACING }}'
```